config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path.yaml

```diff
@@ -18,7 +18,28 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
-        # Copied from config/jobs/kubernetes/sig-testing/verify.yaml.
+        # kind needs /lib/modules and cgroups from the host
```

Comment:
this is in a `reset-kind-volume-mounts: "true"` now, can you add that preset instead?